src/foam/nanos/dig/DUGRule.js

```diff
@@ -0,0 +1,103 @@
+foam.CLASS({
+  package: 'foam.nanos.dig',
+  name: 'DUGRule',
+  extends: 'foam.nanos.ruler.Rule',
+
+  requires: [
+    'foam.nanos.http.Format'
+  ],
+
+ tableColumns: [
+    'id',
+    'documentation',
+    'enabled',
+    'priority',
+    'daoKey',
+    'operation',
+  ],
+
+  searchColumns: [
+    'id',
+    'url',
+    'enabled',
+    'priority',
+    'daoKey',
+    'operation',
+    'after',
+    'validity'
+  ],
+
+  properties: [
+    {
+      name: 'id',
+      visibility: 'RW'
+    },
+    {
+      name: 'ruleGroup',
+      value: 'DUG',
+      hidden: true
+    },
+    {
+      name: 'priority',
+      hidden: true
+    },
+    {
+      class: 'URL',
+      name: 'url',
+      label: 'URL',
+    },
+    {
+      class: 'foam.core.Enum',
+      of: 'foam.nanos.http.Format',
+      name: 'format',
+      view: {
+        class: 'foam.u2.view.ChoiceView',
+        choices: [
+          [2, 'JSON'],
+          [4, 'XML'],
+        ]
+      }
+    },
```

Comment:
The first choice is 'JSON' but if I didn't touch it when creating new DUG rule it didn't get saved to the journal.